onnxruntime/core/providers/openvino/README.md

```diff
@@ -0,0 +1,80 @@
+# Hardware Enabled with OpenVINO Execution Provider
```

Comment:
can you add to docs/execution_providers directory?

Reply:
Will add this